clients/admin-ui/src/features/configure-consent/ConsentManagementModal.tsx

```diff
@@ -0,0 +1,173 @@
+                                <>
+                                  <AccordionButton
+                                    backgroundColor={
+                                      isExpanded ? "gray.50" : "unset"
+                                    }
+                                  >
+                                    <Box flex="1" textAlign="left">
+                                      {purposeName}
+                                    </Box>
+                                    <AccordionIcon />
+                                  </AccordionButton>
+                                  <AccordionPanel backgroundColor="gray.50">
+                                    <Box my={4}>
+                                      <CustomCreatableSelect
+                                        label="Data Uses"
+                                        isMulti
+                                        disableMenu
+                                        isDisabled
+                                        options={[]}
+                                        variant="stacked"
+                                        name={`purposes['${purposeName}'].data_uses`}
+                                      />
+                                    </Box>
+                                    <CustomCreatableSelect
+                                      label="Legal Basis"
+                                      isMulti
+                                      disableMenu
+                                      isDisabled
+                                      options={[]}
+                                      variant="stacked"
+                                      name={`purposes['${purposeName}'].legal_bases`}
+                                    />
+                                  </AccordionPanel>
+                                </>
+                              )}
+                            </AccordionItem>
+                          )
+                        )}
+                      </Accordion>
+                    )}
+                  />
+                  <Box my={4}>
+                    <CustomCreatableSelect
+                      label="Features"
+                      isMulti
+                      options={[]}
+                      disableMenu
+                      isDisabled
+                      variant="stacked"
+                      name="features"
+                    />
+                  </Box>
+                  <CustomCreatableSelect
+                    label="Data Categories"
+                    isMulti
+                    options={[]}
+                    disableMenu
+                    isDisabled
+                    variant="stacked"
+                    name="data_categories"
+                  />
+                </Form>
+              )}
+            </Formik>
+          )}
+        </ModalBody>
+
+        <ModalFooter>
+          <Button onClick={onClose}>Close</Button>
```

Comment:
It might look better if we make the text a bit smaller and switch it to `outline`
```suggestion
          <Button variant="outline" size="sm" onClick={onClose}>
```